mandatory/2-function-creation.js

```diff
@@ -25,9 +37,18 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  // for (let i = 0; i < arrayOfStrings.length; i++) {
+  //   arrayOfStrings[i] = arrayOfStrings[i].trim().toLowerCase().replace("/", "");
+  // }
+  // return arrayOfStrings;
+  let newArray = arrayOfStrings.map(myFunction);
+  return newArray;
+}
 
-/* ======= TESTS - DO NOT MODIFY ===== */
+function myFunction(value, index, array) {
```

Comment:
Not the best name for a function - try to pick a better one :)

Reply:
New name is tidyUpArrayElement :)